crates/matrix-sdk-crypto/Cargo.toml

```diff
@@ -47,6 +47,7 @@ pbkdf2 = { version = "0.12.2", default-features = false }
 rand = { workspace = true }
 rmp-serde = "1.1.1"
 ruma = { workspace = true, features = ["rand", "canonical-json", "unstable-msc3814"] }
+ruma-client-api = { workspace = true, features = ["unstable-msc4048"] }
```

Comment:
I know this should be behind a feature flag, but how do I make a feature flag here enable the `unstable-msc4048` feature flag in Ruma?